core/src/main/java/io/grpc/internal/ClientStream.java

```diff
@@ -46,7 +46,7 @@
    * period until {@link ClientStreamListener#closed} is called. This method is safe to be called
    * at any time and multiple times.
    *
-   * @param reason must be one of Status.CANCELLED or Status.DEADLINE_EXCEEDED
+   * @param reason must have Code CANCELLED, DEADLINE_EXCEEDED, INTERNAL, or UNKNOWN
```

Comment:
@link doesn't work because javadoc cannot reference the fields of an inner class. {@link Status.Code#CANCELLED} didn't work for me.

@code seems fine, but made it harder to read the actual code, even if it makes the generated docs prettier.  @nmittler  would you still like this change?